gr-digital/lib/correlate_access_code_bb_ts_impl.cc

```diff
@@ -90,6 +90,8 @@ unsigned long long correlate_access_code_bb_ts_impl::access_code() const
 inline void correlate_access_code_bb_ts_impl::enter_search()
 {
     d_state = STATE_SYNC_SEARCH;
+    d_data_reg = 0; // K. McQuiggin - zero between access code searches to eliminate false
```

Comment:
The code speaks for itself. Doesn't need signatures.

Resetting data reg also doesn't seem like it should need a comment at all.